static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -47,3 +47,26 @@ The performance of NTFS degrades while handling large volumes of files in a
 directory.
 [Here](https://stackoverflow.com/questions/197162/ntfs-performance-and-large-volumes-of-files-and-directories)
 is the resource for reference.
+
+## Enabling paging with `less`
+
+By default DVC uses `less` to page output for some commands. Although, many
+operating systems have `less` preinstalled, there are still platforms where
+`less` is not installed. This usually happens for Microsoft Windows.
```

Comment:
By default, DVC tries to use [Less](https://github.com/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less) as pager for the output of `dvc pipeline show`. Windows doesn't have the `less` command available however.

Reply:
> use Less as pager

i thought that this link is broken and used https://en.wikipedia.org/wiki/Less_(Unix)
let me know if it's ok

Reply:
Sorry, I meant to use `/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less` as link 🙂

Reply:
isn't that a link to the same section, is it ok?

Reply:
We re in `/doc/command-reference/pipeline/show` here, linking to `/doc/user-guide/running-dvc-on-windows...` Different page in the site 🙂

Reply:
we are rather in `static/docs/user-guide/running-dvc-on-windows.md`? or am i missing something?

Reply:
Oh you're right I was confused! Um I guess this is correct the way you have it actually. I probably copy-pasted the wrong stuff 😅

Reply:
Let me review all links...

Reply:
no worries it happens :) so I'm resolving it.. thanks!